### ISSUE TITLE
SA1516 reports violation between fields if the first one has an attribute

### ISSUE THREAD

**Issue by appel1** [DotNetAnalyzers/StyleCopAnalyzers#1595]
I don't think the following code should cause an SA1516 violation.

```csharp
public class SomeClass
{
  [CanBeNull]
  private Foo foo;
  private bool bar;
}
```

**Comment by sharwell** [comment]
@oatkins @vweijsters Is this a regression from StyleCop "classic"?

**Comment by oatkins** [comment]
Yes. Legacy StyleCop doesn't report anything here.
